### ISSUE TITLE
New translate method breaks redo_lcia

### ISSUE THREAD

**Issue by aleksandra-kim** [brightway-lca/brightway2-calc#2]
**[Original report](https://bitbucket.org/cmutel/brightway2-calc/issue/2) by Chris Mutel (Bitbucket: [cmutel](https://bitbucket.org/cmutel), GitHub: [cmutel](https://github.com/cmutel)).**

----------------------------------------

The `method_filepath` doesn't change, even when the `method` does.

**Comment by aleksandra-kim** [comment]
**Original comment by Chris Mutel (Bitbucket: [cmutel](https://bitbucket.org/cmutel), GitHub: [cmutel](https://github.com/cmutel)).**

----------------------------------------

Bug report from Deborah Sills:

```
#!python

for col, method in enumerate(methods):
    lca.method = method
    lca.load_lcia_data()
    for row, process in enumerate(processes):
        lca.redo_lcia({process: 1})
        results[row, col] = lca.score

```

Probably the `load_lcia_data` function (and similarly the `load_lci_data`) should run `translate` by themselves...

**Comment by aleksandra-kim** [comment]
**Original comment by Chris Mutel (Bitbucket: [cmutel](https://bitbucket.org/cmutel), GitHub: [cmutel](https://github.com/cmutel)).**

----------------------------------------

Fixed in 3bc8892 and 2495873. New syntax will look like:

```
#!python

for col, method in enumerate(methods):
    lca.switch_method(method)
    for row, process in enumerate(processes):
        lca.redo_lcia({process: 1})
        results[row, col] = lca.score
```
